evap/static/scss/components/_tables.scss

```diff
@@ -35,19 +35,8 @@ $table-colors: (
     padding-bottom: 0.4rem;
 }
 
-// Seamless links in tables
-//
-// -1- Expand links to fill the cell
 .table-seamless-links td {
     overflow: hidden;
-
-    a:not(.btn) {
-        color: $black;
-        text-decoration: none;
```

Comment:
Tangential to #2115 -- `text-decoration: none` intended to remove the underlining on hovering (and successfully did that some time ago). We now have no equivalent of that, right?

Reply:
Ah, the results index page doesn't use `.table-seamless-links`. Never mind.